src/mongoose_metrics_definitions.hrl

```diff
@@ -77,6 +77,9 @@
            eval, ?EX_EVAL_SINGLE_VALUE}},
          {nodeUpTime,
           {function, mongoose_metrics, get_up_time, [],
+           tagged, [value]}},
+         {clusterState,
```

Comment:
IMHO, we should report two values here: one for `length(nodes())` and the other one for `length(mnesia:system_info(running_db_nodes))`. It's really hard to say which value is more useful as it depends on situation. 
Also, naming. I'm not really sure what to expected from metric named `clusterState`. What about `nodesDetected` for `length(nodes())` and `nodesConnected` for `length(mnesia:system_info(running_db_nodes))`?
cc @fenek

Reply:
> I'm not really sure what to expected from metric named `clusterState`

Indeed, I'd opt for `clusterSize`, since _state_ is somewhat vague. Anyway, this is a superficial point - whatever it's called won't change its usefulness.

> IMHO, we should report two values here: one for `length(nodes())` and the other one for `length(mnesia:system_info(running_db_nodes))`. It's really hard to say which value is more useful as it depends on situation.

We're not talking about an arbitrary Erlang system, but about MongooseIM - MongooseIM doesn't work properly if Mnesia is not clustered. That was the point of the ticket, let's not change scope once the PR is ready.

To give more context, the operation procedure to tell if Mnesia is clustered is to look at the metric graph and see if all cluster members report the same number for `running_db_nodes`. Specifically, we don't want `mongooseimctl debug` nodes to show up on that metric, which could (I'm not 100% sure, since these might be hidden) if we used `length(nodes())`.
I'm not saying that we should not report `length(nodes())`. I'm saying that I don't think it's as useful and even if it is, we should do it as a separate ticket. This one has already taken a month (sic!).

Reply:
The usefulness of `length(nodes())` comes when we we do have inconsistency in original metric to distinguish between network issues vs Mnesia issues. 
This is however more tricky then I initially expected due to possible not-hidden debug nodes, so I do agree that this should be part of separate story/task. 

+1 for `clusterSize` :)

Reply:
I am for mnesia_running_db_nodes ;)